src/praisonai/praisonai/sandbox/_compat.py

```
@@ -0,0 +1,175 @@
+            else:
+                # Fallback for simple compute providers
+                result = str(await self._compute.run(code))
+                return SandboxResult(
+                    status=SandboxStatus.SUCCESS,
+                    stdout=result,
+                    stderr="",
+                    exit_code=0,
+                    execution_time=0.0,
+                )
+                
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)